src/RemoteFileAPI/MessageHandlers.ts

```diff
@@ -260,4 +260,8 @@ export const RFARequestHandler: Record<string, (message: RFAMessage) => RFAMessa
 
     return new RFAMessage({ result: servers, id: msg.id });
   },
+
+  ping: function (msg: RFAMessage): RFAMessage {
+    return new RFAMessage({ result: {}, id: msg.id });
```

Comment:
```suggestion
    return new RFAMessage({ result: null, id: msg.id });
```
I think null makes more sense for an empty response? And it also makes for a more straightforward type in ResultType.